getting_started/dev-environment/macOS.md

```diff
@@ -0,0 +1,65 @@
+# Instructions for macOS
+
+Install required software:
+- [Homebrew](http://brew.sh/)
+- [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+- [Vagrant](https://www.vagrantup.com/downloads.html)
+
+Install some basic command line tools by running:
+
+```shell
+$ brew install coreutils
+```
+
+Set your PATH to use coreutils:
+
+```shell
+export PATH=/usr/local/opt/coreutils/libexec/gnubin:/usr/local/sbin:$PATH
+export MANPATH=/usr/local/opt/coreutils/libexec/gnuman:$MANPATH
+```
+
+Install the python package manager pip:
+
+```shell
+$ curl -O https://raw.github.com/pypa/pip/master/contrib/get-pip.py
```

Comment:
This is the old URL for get-pip.yml:
```shell
$ curl https://raw.githubusercontent.com/pypa/pip/master/contrib/get-pip.py | python
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   260  100   260    0     0   2222      0 --:--:-- --:--:-- --:--:--  2241
You're using an outdated location for the get-pip.py script, please use the one available from https://bootstrap.pypa.io/get-pip.py
[1]    30234 done       curl https://raw.githubusercontent.com/pypa/pip/master/contrib/get-pip.py | 
       30235 exit 1     python
```

We should change this section to:
```markdown
Install the python package manager pip:

    ```shell
    $ curl https://bootstrap.pypa.io/get-pip.py | python
    ```
```